packages/react/src/utils.ts

```diff
@@ -182,3 +240,38 @@ export class ErrorWrapper extends Error {
     return firstInvalidRecordError?.validationErrors ?? null;
   }
 }
+
+interface QueryPlan {
+  variables: any;
+  query: string;
+}
+
+interface QueryOptions {
+  context?: Partial<OperationContext>;
+  pause?: boolean;
+  requestPolicy?: RequestPolicy;
+  suspense?: boolean;
+}
+
+/** Generate `urql` query argument object, for `useQuery` hook */
+export const useMemoizedQueryArgs = <Plan extends QueryPlan, Options extends QueryOptions>(plan: Plan, options?: Options): UseQueryArgs => {
+  // use a memo as urql rerenders on context identity changes
+  const context = useMemo(() => {
+    return {
+      suspense: !!options?.suspense,
```

Comment:
i'm guessing this keeps suspense off by default for all hooks

Reply:
Correct!